R/medical_detect_entities.R

```diff
@@ -18,9 +19,18 @@ medical_detect_entities <-
 function(
   text,
   language = "en",
+  version = "2",
   ...
 ) {
+    if (version == "1") {
```

Comment:
Please use `match.arg` to match `version` to one of the supported values with a standard error message on unknown values. Also would be a good place for a `switch`.